### DISCUSSION TITLE
Velocity limit

### DISCUSSION THREAD

**Discussion by parilo** [NVlabs/curobo#117] [Software Q&A] [answered]
Is there any way to limit velocity of TCP or joints?

**Comment by balakumar-s** [comment, marked as answer]
Yes, you can set the limit here: https://github.com/NVlabs/curobo/blob/c09d94908d8ad52f557ed59195c1ceb2d0434d65/src/curobo/wrap/reacher/motion_gen.py#L198

It's a scaling factor with 1.0 using the limit from urdf and lower values will scale the limit from the urdf.

Note that we currently cannot change the scale after initialization of a motion generation instance.
